libr/cons/cons.c

```diff
@@ -409,7 +423,7 @@ R_API RCons *r_cons_new() {
 	I.term_raw.c_cflag &= ~(CSIZE|PARENB);
 	I.term_raw.c_cflag |= CS8;
 	I.term_raw.c_cc[VMIN] = 1; // Solaris stuff hehe
-	signal (SIGWINCH, resize);
+	r_sys_sigaction (SIGWINCH, resize, &mask, 0);
```

Comment:
I think here we should use SA_RESTART as flag. This seems to cause the weird behaviour I noticed when changing window size. Also, it may be important to add that flag to other sigaction calls as well...

Reply:
https://www.usna.edu/Users/cs/aviv/classes/ic221/s16/lec/20/lec.html

Reply:
We definitely do not want SA_RESTART for SIGWINCH.

Reply:
kk